transport_versioned_docs/version-1.0.0/workflow-client-server-secure.md

```diff
@@ -0,0 +1,147 @@
+---
+id: jobs
+title: Create jobified client and server
+---
+The unity transport protocol can be configure to encrypt the connection between the server and the client while ensuring the server's/client's authenticity.
+
+## Server authentication
+
+### High level authentication process
+In this configuration, the server will provide a certificate to the client (`rsa`)
+The client will then validate the provided certificate against the one its root certificate (`pem`) and will subsequently validate the server identity.
```

Comment:
Would be more accurate to say that by validating the server's certificate, it is validating its identity. That is, these are not two separate steps.

Reply:
I would also make a note here that the root certificate is also often referred to as a CA certificate here (even if it doesn't actually come from a CA). The term "CA certificate" will be more immediately familiar to more crypto-savvy users.